app/uk/gov/hmrc/fileupload/read/envelope/OutputForTransfer.scala

```diff
@@ -86,24 +84,25 @@ object OutputForTransfer {
   object URLs {
     def envelopesPerDestination(implicit rh: RequestHeader): String = {
       val destination = rh.getQueryString("destination").map(d => s"?destination=$d").getOrElse("")
-      transferRoutes.list().absoluteURL(rh.secure) + destination
+      uk.gov.hmrc.fileupload.controllers.transfer.routes.TransferController.list().absoluteURL(rh.secure) + destination
```

Comment:
don't think we could inject the reverse controllers as @opetch mentioned. Updated to use import alias though